packages/trpc/server/routers/viewer/eventTypes.tsx

```diff
@@ -564,6 +564,12 @@ export const eventTypesRouter = router({
         };
       }
     }
+    // allows unsetting a schedule through { schedule: null, ... }
+    else if (null === schedule) {
```

Comment:
I think in addition to explicitly asking schedule to be set as null, we should set it to null automatically on receiving false value of `useHostSchedulesForTeamEvent`. That should remove the frontend only check for explicitly setting null when `useHostSchedulesForTeamEvent` is false.

Not sure if it has an API call, if yes it should be handled there as well.

Reply:
I had a look but it adds a bit of complexity, ideally we wouldn't have `useHostSchedulesForTeamEvent` at all but this is necessary, without it we can't default to a schedule user schedule with this feature enabled.